winners/router.py

```
@@ -0,0 +1,65 @@
+                    category_name=category_name,
+                    winning_wallet_address_anonymized=anonymize_wallet(draw.winner),
+                    prize_info_summary=prize_summary, # Simplified for now
+                    closed_time=draw.actual_close_time
+                )
+                recent_winners_info.append(winner_info)
+
+        return recent_winners_info
+
+    except PyMongoError as e:
+        print(f"PyMongoError in /winners/recent: {e}")
+        raise HTTPException(status_code=500, detail=f"Database error while fetching recent winners: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)